emu.mk

```diff
@@ -115,6 +115,7 @@ CFLAGS_WARNINGS_EXTRA = \
 
 # Create a variable with the git hash and branch name
 GIT_HASH  = \"$(shell git rev-parse --short HEAD)\"
+GIT_HASH_IS_SHORT = 1
```

Comment:
Don't add unrelated variables. If you're gonna add this, make it do something, like change the length of `GIT_HASH` or something.

Reply:
That was originally the plan, but I realized it wouldn't fit where we have it on the easter egg submenu

Reply:
A PR with "cleanup" in the title usually doesn't add unused code... Clean that up!

Reply:
You are, as usual, entirely correct lol
Cleaning up now